# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libxt/hotfix.patch.cross
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libXt-1.1.4/util/makestrs.c.vanilla	2015-02-17 08:08:44.272000000 +0000
+++ libXt-1.1.4/util/makestrs.c	2015-02-17 08:08:48.124000000 +0000
@@ -27,7 +27,7 @@
 /* Constructs string definitions */
 
 #include <stdio.h>
-#include <X11/Xos.h>
+//#include <X11/Xos.h>
 #include <stdlib.h>
 #include <unistd.h>
 
